Connector Service API upgrade (#2389)
kyma-project/kyma: components/connector-service/README.md

- **internalAPIPort** - This port exposes the Connector Service within Kubernetes cluster. The default port is `8080`.
- **namespace** - Namespace where Connector Service is deployed. The default namespace is `kyma-integration`.
- **tokenLength** - Length of registration tokens. The default value is `64`.
- **appTokenExpirationMinutes** - Time after which tokens for applications expire and are no longer valid. The default value is `5` minutes.
- **runtimeTokenExpirationMinutes** - Time after which tokens for runtimes expire and are no longer valid. The default value is `10` minutes.

> - **caSecretName** - Name of the secret which contains root CA. The default value if `nginx-auth-ca`

Reviewer: Done

> - **caSecretName** - Name of the secret which contains root CA. The default value is `nginx-auth-ca`

Reviewer 2: ```suggestion
- **caSecretName** - Name of the secret which contains the root Certification Authority (CA). The default value is `nginx-auth-ca`.
```

> - **requestLogging** - Flag for logging incoming requests. `False` by default.

Reviewer 2: ```suggestion
- **requestLogging** - Flag for logging incoming requests. It is set to `False` by default.
```

> - **connectorServiceHost** - Host at which this service is accessible, used for URL generating. Default host is `cert-service.wormhole.cluster.kyma.cx`.

Reviewer 2: ```suggestion
- **connectorServiceHost** - Host under which this service is accessible. It is used for generating the URL. The default host is `cert-service.wormhole.cluster.kyma.cx`.
```

> - **appRegistryHost** - Host at which this Application Registry is accessible. Default value is empty string.

Reviewer 2: ```suggestion
- **appRegistryHost** - Host under which this Application Registry is accessible. The default value is an empty string.
```

> - **eventsHost** - Host at which this Event Service is accessible. Default value is empty string.

Reviewer 2: ```suggestion
- **eventsHost** - Host under which this Event Service is accessible. The default value is an empty string.
```

> - **getInfoURL** - URL at which management information is available. If not provided it will be constructed based on `connectorServiceHost`.

Reviewer 2: ```suggestion
- **getInfoURL** - URL at which the management information is available. If not provided, it bases on `connectorServiceHost`.
```

> - **group** - Group for which certificates will be generated. If not provided, it will be required in the request header to token endpoint.

Reviewer 2: ```suggestion
- **group** - Group for which certificates are generated. If the chart does not provide the default value, you must specify it in the request header to the token endpoint.
```

> - **tenant** - Tenant for which certificates will be generated. If not provided, it will be required in the request header to token endpoint.

Reviewer 2: ```suggestion
- **tenant** - Tenant for which certificates are generated. If the chart does not provide the default value, you must specify it in the request header to the token endpoint.
```
